src/services/db/models.py

```diff
@@ -31,6 +31,7 @@ class CatalogueItem(db.Model):
     checksum_algorithm = db.Column(db.String)
     checksum_value = db.Column(db.String)
 
+    transfer_id = db.Column(db.String)
```

Comment:
Is `transfer_id` required? (I might be out of context). Isn't file uuid feasible to do the tracking?

Reply:
Edit: Got it....